### ISSUE TITLE
NECB 2017 : Envelope Compliance

### ISSUE THREAD

**Issue by phylroy** [canmet-energy/btap_tasks#139]
### Description
This task is to implement the NECB 2017 Envelope as part of the standards work. This will create a new method inheritiing from NECB 2015 to modify all construction in a model to comply with NECB 2017 Section 4 rules w.r.t. energy simulation. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/canmet-energy/btap-development-environment
https://github.com/canmet-energy/docker-openstudio
https://github.com/canmet-energy/btap
https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Codes you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez

**Comment by phylroy** [comment]
Created branch nrcan_139 to work on task.
